### ISSUE TITLE
[Question] Headless chrome

### ISSUE THREAD

**Issue by MartyJustice** [codeborne/selenide#566]
Hi all,

I would like to use selenide with headless chrome. I'm using a custom webdriver with these arguments added:
```
options.addArguments("--headless");
		options.addArguments("--disable-gpu"); 
```
but I get the following error:

``
Starting ChromeDriver 2.30.477691 (6ee44a7247c639c0703f291d320bdf05c1531b57) on port 28467
Only local connections are allowed.
Jun 30, 2017 11:28:06 AM com.codeborne.selenide.webdriver.WebDriverFactory adjustBrowserSize
WARNING: Cannot maximize ChromeDriver: org.openqa.selenium.WebDriverException: unknown error: cannot get automation extension
from unknown error: page could not be found: chrome-extension://aapnijgdinlhnhlmodcfapnahmbfebeb/_generated_background_page.html
  (Session info: headless chrome=59.0.3071.86)
  (Driver info: chromedriver=2.30.477691 (6ee44a7247c639c0703f291d320bdf05c1531b57),platform=Linux 4.4.0-79-generic x86_64) (WARNING: The server did not provide any stacktrace information)
``
Chrome 59.0.3071.86
Chromedriver: 2.30.477691

I'm still on an old version of Selenide (3.7), but I don't know if this matters. 

Anybody know what's going on?

**Comment by asolntsev** [comment]
@MartyJustice Sorry for the long replay. Unfortunately I haven't experimented with headless chrome yet. So I cannot answer your question. :( 
I hope I will investigate it soon.

**Comment by SergeyPirogov** [comment]
First of all, you are using a wrong way to add arguments. The correct one is
```
options.addArguments("headless","disable-gpu")
```
Second, headless doesn't support maximize command, so to use headless Chrome with Selenide
you should set Configuration.startMaximized= false

This should work well

**Comment by MartyJustice** [comment]
@asolntsev  @SergeyPirogov  
Thanks for the response. 

The error is fixed by adding 
Configuration.startMaximized= false to the configuration (What @SergeyPirogov said)
This turned out to be unrelated to the headless chrome issues I experienced. 

I turned out headless chrome and this piece of code do not like each other:
getWebDriver().manage().window().setSize(new Dimension(x, y));

I now set a browser size via chrome options and headless chrome is working smoothly.
